java-logging-appinsights/src/main/resources/logback-includes.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<included>
+    <appender name="APP_INSIGHTS" class="com.microsoft.applicationinsights.logback.ApplicationInsightsAppender">
```

Comment:
does this mean there needs to be a docs update to say you need to set your ROOT_APPENDER to APP_INSIGHTS?

Reply:
no, this appender is attached to **all** `uk.gov.hmcts` loggers

in certain situations, if appender reference is not found in the configuration (suppose you use APP_INSIGHTS in root) the application might fail to start. i think particularly if you temper with joran configuration manually.

- if added in root - there will be an error of missing APP_INSIGHTS appender
- if left like this - no error and you are free to use whichever java-logging library version and module. downside - hardcoded to `uk.gov.hmcts`